### DISCUSSION TITLE
Кнопка скрытия боковой панели комментариев

### DISCUSSION THREAD

**Discussion by kabalin** [PastVu/pastvu#659] [Ideas / Идеи и предложения]
Предлагается добавить кнопку для скрытия панели комментариев (при необходимости). Это было бы удобно для небольших экранов.

https://github.com/PastVu/pastvu/assets/329780/a98e1f57-c990-44ea-a76f-82584e4b6033

**Comment by mkgrgis** [comment]
Главное чтобы на небольшом экране можно было надёжно попасть на кнопку без стилуса.

**Reply by kabalin** [reply]
Это да, но это часть общей проблемы mobile-friendly. Кнопка в данном случае будет в едином стиле с существующими.

**Comment by kabalin** [comment]
Добавлено в [2.0.44](https://github.com/PastVu/pastvu/pull/673)
